lib/bugsnag/api/error.rb

```diff
@@ -59,7 +57,9 @@ def data
       end
 
       def response_error
```

Comment:
Alternatives which break complete backwards compat for message parsing would be:
```
errors = Array(data[:errors] || data[:error])
"Errors: #{errors}" if errors.any?
```
or
```
errors = data[:errors] || (data[:error] ? [data[:error]] : nil)
"Errors: #{errors}" if errors.any?
```